examples/PhasorDynamics/Small/TenGen/TenGen_GenClassical/CMakeLists.txt

```
@@ -1,12 +1,12 @@
-add_executable(phasordynamics_example3_genclassical example3_genclassical.cpp)
-target_link_libraries(phasordynamics_example3_genclassical
+add_executable(phasordynamics_example_tengen_genclassical Example_TenGen_GenClassical.cpp)
+target_link_libraries(phasordynamics_example_tengen_genclassical
 	GRIDKIT::phasor_dynamics_bus 
 	GRIDKIT::phasor_dynamics_bus_fault 
 	GRIDKIT::phasor_dynamics_branch
 	GRIDKIT::phasor_dynamics_gen_classical
 	GRIDKIT::phasor_dynamics_load
 	GRIDKIT::solvers_dyn)
-install(TARGETS phasordynamics_example3_genclassical RUNTIME DESTINATION bin)
+install(TARGETS phasordynamics_example_tengen_genclassical RUNTIME DESTINATION bin)
 
```

By the same measure, I think we can shorten target names without losing information:
```suggestion
install(TARGETS phasordynamics_tengen_classical RUNTIME DESTINATION bin)
```
The entire directory is with example only, so I would drop "example" from example names.

---

Similar applies to a few other places in the PR.